docker-compose.yml

```
@@ -46,3 +46,53 @@ services:
 
 networks:
   twotier:
+
+
+# version: "3.8"
+
+# services:
+#   mysql:
+#     user: "${UID}:${GID}" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)